### PR TITLE
nits in e2e test

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

Expect(err).NotTo(HaveOccurred()) has been called in:

https://github.com/kubernetes/kubernetes/blob/2fdd328d057442ad9848d4798f955c11f8bf858e/test/e2e/storage/vsphere/pvc_label_selector.go#L83-L84

remove unused this in:
https://github.com/kubernetes/kubernetes/blob/2fdd328d057442ad9848d4798f955c11f8bf858e/test/e2e/storage/vsphere/pvc_label_selector.go#L110
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/storage/vsphere/pvc_label_selector.go [hunk 1]
```diff
@@ -105,9 +105,8 @@ var _ = utils.SIGDescribe("PersistentVolumes [Feature:LabelSelector]", func() {
 })
 
 func testSetupVSpherePVClabelselector(c clientset.Interface, nodeInfo *NodeInfo, ns string, ssdlabels map[string]string, vvollabels map[string]string) (volumePath string, pv_ssd *v1.PersistentVolume, pvc_ssd *v1.PersistentVolumeClaim, pvc_vvol *v1.PersistentVolumeClaim, err error) {
```
Comment (mariantalla) [inline test/e2e/storage/vsphere/pvc_label_selector.go]: Does err need to be passed in as an argument at all here? It seems to be immediately overwritten by `nodeInfo.VSphere.CreateVolume(...)`.

cc @hoegaarden

Comment (tanshanshan) [reply test/e2e/storage/vsphere/pvc_label_selector.go]: I think so,  err  is redundant ,I will remove it

Comment (tanshanshan) [reply test/e2e/storage/vsphere/pvc_label_selector.go]: error is as a  return value , I think no need to do anything

Comment (oomichi) [reply test/e2e/storage/vsphere/pvc_label_selector.go]: I agree with tanshanshan here.
That seems we forgot to remove the Expect(err) in https://github.com/kubernetes/kubernetes/commit/c0490fa62365a37d1a4c3c27b746822782849fc2#diff-0714d41e8eaadd3eb93bba91a7347782R114

**Review by oomichi — COMMENTED** [review]
/lgtm

**Comment by oomichi** [comment]
For getting approval

/assign @spiffxp

**Pull request merged**